### ISSUE TITLE
Expose identity of edges in the dashboard

### ISSUE THREAD

**Issue by scottcarol** [linkerd/linkerd2#2709]
### What problem are you trying to solve?

See #2708

### How should the problem be solved?

For dashboard users, exposing identity of edges in an "Identity" table. In the future, there could be a dedicated "Identity" view page, but initially this feature will be only a table.

### Any alternatives you've considered?

See #2708

### How would users interact with this feature?

An `Identity` table will appear in the `Overview` view of the dashboard for each namespace, and in every resource detail view. 

The table will contain these columns (as depicted above and in the CLI as `linkerd edges identity`):
    - SRC
    - DST
    - Client
    - Server
    - Message/Reason (We will experiment with the best way to show this for readability/usefulness. Exposing the entire reason may be hard to parse, but obscuring it in favor of a tooltip creates more work for the user.)

![grambp](https://user-images.githubusercontent.com/2289389/56306199-c41d8100-60f6-11e9-8369-a9785b3c667e.png)

This issue builds off ideas in #2605 and #2606.

**Comment by scottcarol** [comment]
@grampelberg here’s what I have so far for the Edges table.

The default `Overview` view is by namespace, but requests to the `edges` API endpoint must specify one of 6 resource types:
- daemonsets
- deployments
- jobs
- pods
- replicationcontrollers
- statefulsets

In the below screenshot from the `Overview` view for the `emojivoto` namespace, I query for pod edges and include a column for resource type. Alternatively I could make multiple queries and pull in all available edges. 

Or, I could only have the Edges table only in the views for the 6 resource types specified above. What do you think?

![Screen Shot 2019-06-12 at 9 31 56 AM](https://user-images.githubusercontent.com/2289389/59370515-881e2b00-8cf7-11e9-912f-799246ef1064.png)

**Comment by grampelberg** [comment]
Let's add the edges table to each of the resource detail views and only there for now. WDYT? 

What do you think about opening up two new issues? The overview page needs some love as it is really deployment based right now and should be more general. The resource list pages are also a little questionably valuable (though the namespace one is super useful and buried).

**Comment by scottcarol** [comment]
I think adding the edges table to the resource detail views is a good plan. In that case I'll omit the column specifying resource name.

Re: two new issues, one issue is adding the edges table to each resource detail view, and one issue is rethinking the overview page to be more general/helpful? Yes I can do that.

Related question: educating users on the term "Edges" and how they could use the Edges table for debugging. Would a <img width="10" alt="Screen Shot 2019-06-12 at 10 59 30 AM" src="https://user-images.githubusercontent.com/2289389/59374676-43978d00-8d01-11e9-824d-cd6578eb520b.png"> next to the table title with some text on hover would be useful?

**Comment by grampelberg** [comment]
I love tooltips!

**Comment by scottcarol** [comment]
@grampelberg about to open 2 new issues, one for the Edges table in the dashboard resource detail views, and one to rethink the `overview` page.

Clarification re: resource detail view. There are two options: adding to 
`http://localhost:7777/pods` or adding the table to the specific pod:
`http://localhost:7777/namespaces/emojivoto/pods/emoji-646ddcc5f9-hl7s7`

For `/pods/`, because that view shows all pods in each namespace, the result would be a table with edges from multiple namespaces.

If we display the Edges table on the resource detail view, ex. `/namespaces/emojivoto/pods/emoji-646ddcc5f9-hl7s7`, we can do a simple query for all pods in that namespace, but the edges returned might not include `emoji-646ddcc5f9-hl7s7` as a `SRC` or `DST`. So we might end up with an Edges table that doesn't include the specific resource we're looking at. I'm fine with this because it could be valuable information for the user to know that the resource is NOT part of an edge, and prefer this option. Thoughts?

**Comment by grampelberg** [comment]
I think of `/pods` as a resource list view and `/namespaces/emojivoto/pods/emoji-646ddcc5f9-hl7s7` as a resource detail view.

The table really should only show edges that contain the resource in question.
